core/network/address.go

```diff
@@ -777,7 +777,7 @@ func DeriveAddressType(value string) AddressType {
 
 // ScopeMatchPublic is an address scope matching function for determining the
 // extent to which the input address' scope satisfies a requirement for public
-// accessibility.
+// accessibility. Prefers IPv4 over IPv6.
 func ScopeMatchPublic(addr Address) ScopeMatch {
```

Comment:
IMO we should rename this one to `ScopeMatchPublicPreferIPv4` and have the preferred one as `ScopeMatchPublic`

Reply:
I agreed with this. The func name should imply the bias.

Reply:
This 100%.